### ISSUE TITLE
property 'code' of undefined in node_modules/marked/lib/marked.js:1226

### ISSUE THREAD

**Issue by GabLeRoux** [tldr-pages/tldr#340]
Command:
```
tldr git
```

Result:
```
/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1226
    throw e;
    ^

TypeError: Cannot set property 'code' of undefined
Please report this to https://github.com/chjj/marked.
    at Renderer.r.codespan (/usr/local/lib/node_modules/tldr/lib/parser.js:47:32)
    at InlineLexer.output (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:664:28)
    at Parser.tok (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1075:50)
    at Parser.tok (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1033:22)
    at Parser.parse (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:935:17)
    at Function.Parser.parse (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:922:17)
    at marked (/usr/local/lib/node_modules/tldr/node_modules/marked/lib/marked.js:1218:19)
    at Object.exports.parse (/usr/local/lib/node_modules/tldr/lib/parser.js:50:3)
    at /usr/local/lib/node_modules/tldr/lib/tldr.js:89:23
    at /usr/local/lib/node_modules/tldr/lib/cache.js:22:5
```

Only happened to me with `tldr git`, the others I've tried seemed ok.

```
tldr --version
1.0.1
```

**Comment by GabLeRoux** [comment]
confirmed it works after running `tldr update`
